FEMComputationalFluidDynamicsAnalysis/postProcessing/computeResultingTotalForceOnSelectedDomain.m

```diff
@@ -24,16 +24,11 @@
 %
 %% Function main body
     
-% Reshape the vector to [Vx, Vy, p]
-FComplete = -reshape(FComplete, [analysis.noFields,                     ...
-             length(FComplete) / analysis.noFields])';
-
-% Select only the forces on the domain
-Force = FComplete(nodesDomain, 1:analysis.noSpatialDimensions);
-
 F = zeros(analysis.noSpatialDimensions,1);
+DOFsDomain = analysis.noFields * nodesDomain - analysis.noSpatialDimensions;
 for k = 1:analysis.noSpatialDimensions
-    F(k,1) = sum(Force(:,k)) * parameters.rho;
+    activeDOFsDomain = DOFsDomain + (k-1);
```

Comment:
You are adding a scalar to a vector here, this does not seem very similar to what I proposed.